wgsl/index.bs

```diff
@@ -6887,6 +6888,13 @@ That's not a full user-defined function declaration.
     <td>Test a normal value according to [[!IEEE-754|IEEE-754]].<br>
     [=Component-wise=] when |T| is a vector.
 
+  <tr algorithm="fixed-sized array length">
+    <td>|e|: ptr&lt;storage,array&lt;|T|,|N|&gt;&gt;
```

Comment:
Copy-paste bug, should have a parameter for the storage class.

Reply:
Thanks. Fixed now.